### DISCUSSION TITLE
RESULT_CODE_KILLED_BAD_MESSAGE

### DISCUSSION THREAD

**Discussion by OneSeven** [wailsapp/wails#992] [Q&A]
Running the following code will crash, is it related to wails?
Irrelevant, I will report to Microsoft

```js
let mediaSourc = new window.MediaSource();
let mediaSourceObjectURL = window.URL.createObjectURL(mediaSourc);
```

@leaanthony

**Comment by leaanthony** [comment]
Pure JS issue = best ask Microsoft

**Reply by OneSeven** [reply]
Hope to help rule out,
Now Microsoft can't determine the cause
https://github.com/MicrosoftEdge/WebView2Feedback/issues/1968
